src/_data/LiveData.js

```diff
@@ -40,17 +47,18 @@ const subscribeToWatchlist = () => {
 };
 
 export const initUnauthorized = () => {
-    api.getActiveSymbolsFull();
     api.getTradingTimes();
     api.getAssetIndex();
 
     readNewsFeed().then(articles => api.events.emit('news', articles));
     getVideosFromPlayList().then(videos => api.events.emit('videos', videos));
+    subscribeToSelectedSymbol();
```

Comment:
This should not happen here.
LiveData should have no idea what is selected symbol or that we even use immutable.js :dancer: